### PR TITLE
Implementations: Fix Typo 

### PR DESCRIPTION
**Pull request by elazarg**

### CODE REVIEW

File: _implementors/servers.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ index: 1
 | C/C++/Objective-C | [Jacob Dufault](https://github.com/jacobdufault), [MaskRay](https://github.com/MaskRay), [topisani](https://github.com/topisani) | [cquery](https://github.com/jacobdufault/cquery) |
 | CSS/LESS/SASS | MS | [vscode-css-languageservice](https://github.com/Microsoft/vscode-css-languageservice) |
 | [Ceylon](https://ceylon-lang.org/)| [John Vasileff](https://github.com/jvasileff) | [vscode-ceylon](https://github.com/jvasileff/vscode-ceylon) |
-| [Clojure](https://clojure.org/)] [Case Nelson](https://github.com/snoe) | [clojure-lsp](https://github.com/snoe/clojure-lsp) |
+| [Clojure](https://clojure.org/)] | [Case Nelson](https://github.com/snoe) | [clojure-lsp](https://github.com/snoe/clojure-lsp) |
```
Comment (elazarg) [inline _implementors/servers.md]: Oops - the closing square bracket shouldn't be there either.

Comment (dbaeumer) [reply _implementors/servers.md]: I checked the file and it looks good to me. What do I miss?

Comment (elazarg) [reply _implementors/servers.md]: `[Clojure](https://clojure.org/)]` should be `[Clojure](https://clojure.org/)`

Comment (dbaeumer) [reply _implementors/servers.md]: Thanks. Fixed.

**Pull request merged**